app/src/main/java/com/sopt/dive/MainActivity.kt

```diff
@@ -0,0 +1,29 @@
+package com.sopt.dive
+
+import android.os.Bundle
+import androidx.activity.ComponentActivity
+import androidx.activity.compose.setContent
+import androidx.activity.enableEdgeToEdge
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.material3.Scaffold
+import androidx.compose.ui.Modifier
+import androidx.navigation.compose.rememberNavController
+import com.sopt.dive.navigation.LoginNavGraph
+
+class MainActivity : ComponentActivity() {
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        enableEdgeToEdge()
+        setContent {
+                Scaffold(modifier = Modifier.Companion.fillMaxSize()) { innerPadding ->
+
+                    val navController = rememberNavController()
+
+                    LoginNavGraph(
+                        navController = navController,
+                        innerPadding = innerPadding
+                    )
+                }
+            }
```

Comment:
Scaffold 안에서 navController 를 선언하기 보다는 Scaffold 밖에서 선언하는게 더 좋을거 같아요
Scaffold 안에 두면 범위가 불필요하게 줄어들고 만약 Scaffold 에서 topBar 등과 같이 선언하게 되면 이를 가져다 쓰기에도 편합니다 !

```suggestion
        setContent {
                val navController = rememberNavController()
                Scaffold(
                    modifier = Modifier.Companion.fillMaxSize(),
                     //topBar = { DiveTopBar(onBack = {navController.popBackStack() }) },
                ) { innerPadding ->

                    LoginNavGraph(
                        navController = navController,
                        innerPadding = innerPadding
                    )
                }
            }
```